### DISCUSSION TITLE
String.TrimStart(char trimChar, int maxChars)

### DISCUSSION THREAD

**Discussion by UpperAvenue** [dotnet/runtime#74818] [Ideas]
I would like to propose an overload to the TrimStart method that limits the number of characters trimmed.  Additionally similar overloads for the TrimEnd method.

For example,
```
"[[[Name], Welcome to [City]]]".TrimStart('[', 2).TrimEnd(']', 2) == "[Name], Welcome to [City]"
```

**Comment by LeaFrock** [comment]
It's easy to make your own extension methods, and this scenario is not very common. 

IMO it's not worthy to be added into BCL.
